src/uct/cuda/cuda_ipc/cuda_ipc_iface.h

```diff
@@ -16,23 +17,18 @@
 #include "cuda_ipc_cache.h"
 
 
-#define UCT_CUDA_IPC_MAX_PEERS  16
+KHASH_MAP_INIT_INT(cuda_ipc_queue_desc, uct_cuda_queue_desc_t*);
```

Comment:
```suggestion
KHASH_MAP_INIT_INT(cuda_ipc_queue_desc, uct_cuda_queue_desc_t);
```
then you do not need to malloc/free hash values

Reply:
Will consider making this change.

Reply:
Fixed in https://github.com/openucx/ucx/pull/10538